docs/how-to-guides/themes/global-settings-and-styles.md

```diff
@@ -321,6 +321,7 @@ To retain backward compatibility, the existing `add_theme_support` declarations
 | `editor-color-palette`      | Provide the list of colors via `color.palette`.           |
 | `editor-font-sizes`         | Provide the list of font size via `typography.fontSizes`. |
 | `editor-gradient-presets`   | Provide the list of gradients via `color.gradients`.      |
+| `editor-spacing-sizes`      | Provide the list of font size via `typography.fontSizes`. |
```

Comment:
Minor copy/paste nit:

```suggestion
| `editor-spacing-sizes`      | Provide the list of spacing sizes via `spacing.spacingSizes`. |
```

Reply:
Fixed in 2dcd295fc7eb2022ba1a7dd000c150b34f901fbd